src/Errors/EleventyErrorHandler.js

```diff
@@ -77,7 +76,7 @@ class EleventyErrorHandler {
 	//https://nodejs.org/api/process.html
 	log(e, type = "log", chalkColor = "", forceToConsole = false) {
 		if (process.env.DEBUG) {
-			debug("Full error object: %o", util.inspect(e, { showHidden: false, depth: null }));
+			debug("Full error object: %O", e);
```

Comment:
This change is not quite the same, right? https://github.com/debug-js/debug/blob/7e3814cc603bf64fdd69e714e0cf5611ec31f43b/src/node.js#L262 looks like it uses util.inspect defaults, which is `depth: 2`

https://nodejs.org/docs/latest/api/util.html#utilinspectobject-showhidden-depth-colors

Reply:
Gonna roll forward without this line!